docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-    - Updating an application using the method PUT replaces all existing objects, including any persistent storage volumes, of the application.
-    - Scaling down or redistributing an application that uses persistent storage volumes might delete existing persistent volumes along with pod instances.
\ No newline at end of file
+If your application is distributed per region, a dialog box similar to the following appears.
+
+<<new_Redistribute-Your-Application-Distribute-per-Region.png goes here>>
+
+1. Under **Targets**, accept or change the regions and replicas per region shown for existing targets.
+2. To add targets, click the **+ Add Target** button. When a new row appears at the bottom of **Targets**, select a region and enter the number of replicas for that region.
+3. To remove a target, click the trash can icon at the right side of the target.
+4. When you finish, click **Redistribute**.
+
+###Applications Distributed per Pop
+
+If your application is distributed per PoP, a dialog box similar to the following appears.
+
+<<new_Redistribute-Your-Application-Distribute-per-Pop.png goes here>>
+
+1. Under **Targets**, accept or change the PoPs and replicas per PoP shown for existing PoPs.
+2. To add targets, click the **+ Add Target** button. When a new row appears at the bottom of **Targets**, select a PoP and enter the number of replicas for that PoP.
+3. To remove a target, click the trash can icon at the right side of the target.
+4. When you finish, click **Redistribute**.
+
+##Setting the Container 
+If you select **Set container image** from the Quick Update drop-down list, a Set Container Image dialog box similar to the following appears. 
+
+<<new_Set-Container-Image-Dialog-Box.png goes here>>
+
+**Note:** This operation might cause your existing pods to be deleted and recreated.
+
+1. To change the Docker image name, replace the name in the **Image** field. Typically, this name consists of docker image registry, image, and tag.
+<br><ul>- If you omit the registry, the system defaults to docker hub.</ul><ul>- If you omit the tag, the system defaults to "latest".</ul><ul>- If your entry is not valid, the image pulling and application deployment fails.</ul></br>
+2. To change the image pull secret, drag the <strong>Change Image Pull Secret</strong> control to the right. Then click in the field displaying the current secret and select an image pull secret.
+3. To add an image pull secret, drag the <strong>Change Image Pull Secret</strong> control to the right. Click in the field displaying the current secret and select <strong>+Add Image Pull Secret</strong>. Complete the fields in the Add ImagePullSecret dialog box, and then click <strong>Submit</strong>. <br>
+4. When you finish, click **Update**.
+   
+##Upgrading Resources
+If you select **Upgrade resources** from the Quick Update drop-down list, a Set Container Image dialog box similar to the following appears. 
```

Comment:
"Set Container Image" here isn't correct.

Reply:
Fixed.

Reply:
I confirm the text is correct now. Find a new issue. The screenshot below the sentence doesnot exist.
![image](https://user-images.githubusercontent.com/43203744/90623029-8976ac00-e248-11ea-9b56-a4267fbad9b4.png)

Reply:
Fixed